core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/WriteOperationParserSuite.scala

```diff
@@ -180,80 +421,107 @@ class WriteOperationParserSuite extends FunSuite {
       expectedOutputColumns = StringUtils.UNKNOWN_EXTRACT,
       expectedWriteMode = StringUtils.UNKNOWN_EXTRACT,
       expectedTableName = StringUtils.UNKNOWN_EXTRACT,
-      expectedDatabaseName = StringUtils.UNKNOWN_EXTRACT
+      expectedDatabaseName = StringUtils.UNKNOWN_EXTRACT,
+      expectedPartitionCols = StringUtils.UNKNOWN_EXTRACT
     )
   }
 
-  test("InsertIntoHadoopFsRelationCommand - Empty output columns") {
-    val node = new SparkPlanGraphNode(
+  test("InsertIntoHiveTable cmd — No catalog prefix") {
```

Comment:
This unit test seems to take an unusual time. Is this expected? 

![image](https://github.com/user-attachments/assets/ad3333e8-fa7b-4686-b7c6-775d8c050196)

Reply:
I guess it is a GC that triggered in the VM.
It does not take more than hundreds milliseconds.

Reply:
Since this test was taking a larger time everytime I was running the test, I thought it is a deterministic case.